barretenberg/cpp/src/barretenberg/bb/main.cpp

```diff
@@ -1311,7 +1311,7 @@ void prove_honk_output_all(const std::string& bytecodePath,
     std::string proofFieldsPath = outputPath + "/proof_fields.json";
 
     VerificationKey vk(
-        prover.instance->proving_key); // uses a partial form of the proving key which only has precomputed entities
+        prover.proving_key->proving_key); // uses a partial form of the proving key which only has precomputed entities
```

Comment:
what's the plan here? Is it to move everything from the proving key to the decider proving key?

Reply:
Yes, Honk proving key is not useful, @ledwards2225  agrees

Reply:
I guess I would disagree, but it's fine that we're merging them

Reply:
I would also disagree about the "absurd-looking proving_key->proving_key". That's just a result of us having sub-provers, like the oink prover being part of the ultra prover/folding prover.